### PR TITLE
Add toggle component and test

### PR DESCRIPTION
**Pull request by jiggsaw85**
**Jira task:** [https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?selectedIssue=V5GA-8](https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?selectedIssue=V5GA-8)

### CODE REVIEW

File: src/components/form/SwitchButton.vue [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+      />
+      <span
+        class="toggle-indicator"
+        :class="[backgroundStyles, hover]"
+        :style="indicatorStyles"
+        @mouseenter="hover = 'hover'"
+        @mouseleave="hover = ''"
+      />
+    </span>
+    <span v-if="label" class="label">{{ label }}</span>
```
[the remark this answers is not in the document]
Comment (jiggsaw85) [reply src/components/form/SwitchButton.vue]: @Aviv-Galmidi All fixed per comments.

File: src/views/help/HelpAndSupport.vue [hunk 2]
```diff
@@ -53,15 +53,20 @@
...
 
 <script lang="ts">
 import {defineComponent} from 'vue'
+import FormToggle from '@/components/FormToggle.vue'
 
 export default defineComponent({
   name: 'HelpAndSupport',
+  components: {
+    FormToggle,
+  },
```
Comment (Aviv-Galmidi) [inline src/views/help/HelpAndSupport.vue:69]: Please remove the component usage from this page

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

**Review by Aviv-Galmidi — APPROVED** [review]

**Comment by Aviv-Galmidi** [comment]
Please resolve conflicts and then we'll merge

**Comment by jiggsaw85** [comment]
Conflicts fixed.

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**